src/test/java/ru/training/at/hw3/main/differentelementspage/DifferentElementsPage.java

```diff
@@ -0,0 +1,60 @@
+package ru.training.at.hw3.main.differentelementspage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+public class DifferentElementsPage {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @FindBy(xpath = "//label[1]/input")
+    WebElement waterCheckBox;
```

Comment:
If you want to find concrete element and if you know its text you could use it in XPath - //label[contains(., 'Water')]/input
Think about that and fix all similar usages in project.